orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/monitoreddeploy/EvaluateDeploymentHealthTaskSpec.groovy

```diff
@@ -45,7 +52,7 @@ class EvaluateDeploymentHealthTaskSpec extends Specification {
     given:
```

Comment:
though underlying exceptions at present are retrofit, wouldn't it be meaningful to change  "retrofit errors" to SpinnakerServerExceptions in the test name?